### DISCUSSION TITLE
Release 22.0.6 does not exists.

### DISCUSSION THREAD

**Discussion by alokrbl** [keycloak/keycloak#25468] [Archive]
I am unable to download 22.0.6 distribution 

steps - open link -> https://github.com/keycloak/keycloak/releases/download/22.0.6/keycloak-22.0.6.tar.gz
Response - Not Found.

I am using dependency manager to find the latest version of the release. There is an issue because tag for 22.0.6 exists but distribution to download does not exists

Pls help as i need latest version from 22 which is 22.0.6.  Pl note 22.0.5 is ok.

Thanks
Alok

**Comment by jonkoops** [comment]
This release does not exist, and that is on purpose. If you want the latest version of Keycloak you'll have to upgrade to 23.
